install/0000_00_cluster-version-operator_01_admingate_configmap.yaml

```diff
@@ -1,5 +1,8 @@
 apiVersion: v1
 kind: ConfigMap
+data:
+ack-4.12-kube-1.26-api-removals-in-4.13: |-
```

Comment:
This is missing a two-space indent, resulting in it being a `data` sibling instead of the `data` child we want:

```console
$ git cat-file -p origin/pr/908:install/0000_00_cluster-version-operator_01_admingate_configmap.yaml | yaml2json 
{
  "apiVersion": "v1",
  "kind": "ConfigMap",
  "data": null,
  "ack-4.12-kube-1.26-api-removals-in-4.13": "Kubernetes 1.26 and therefore OpenShift 4.13 remove several APIs which require admin consideration. Please see the knowledge article https://access.redhat.com/articles/6958394 for details and instructions.",
  "metadata": {
...
```

Reply:
Fixed it.